ts/indicators/cci.src.ts

```diff
@@ -111,12 +111,12 @@ seriesType<Highcharts.CCIIndicator>(
      */
     {
         getValues: function (
-            series: Highcharts.CCIIndicator,
+            series: Highcharts.Series,
             params: Highcharts.CCIIndicatorParamsOptions
         ): (boolean|Highcharts.IndicatorValuesObject) {
             var period: number = (params.period as any),
                 xVal: Array<number> = (series.xData as any),
-                yVal: Array<Array<number>> = series.yData,
+                yVal: Array<Array<number>> = (series.yData as any),
```

Comment:
Additionally the `CCIIndicator.yData` property should be removed in the class declaration.